test/fail_compilation/fail13902.d

```diff
@@ -57,7 +57,7 @@ fail_compilation/fail13902.d(88): Error: Using the result of a comma expression
 fail_compilation/fail13902.d(75): Error: returning `& x` escapes a reference to parameter `x`
 fail_compilation/fail13902.d(76): Error: returning `&s1.v` escapes a reference to parameter `s1`
 fail_compilation/fail13902.d(81): Error: returning `& sa1` escapes a reference to parameter `sa1`
-fail_compilation/fail13902.d(82): Error: returning `&sa2[0][0]` escapes a reference to parameter `sa2`
```

Comment:
This PR introduces a regression in D error messages.
```D
int* f()
{
    int[2][2] sa2;
    return &sa2[1][1];
}
```

> returning `(& sa2+12)` escapes a reference to local variable `sa2`

That's not what's being returned.

Reply:
yes, it is what is being returned.